packages/api/endpoints/granules.js

```diff
@@ -124,7 +124,22 @@ const create = async (req, res) => {
       })
     ) {
       return res.boom.conflict(
-        `A granule already exists for granule_id: ${granule.granuleId}`
+        `A granule already exists for granule_id: ${granule.granuleId} with collection ID ${granule.collectionId}`
```

Comment:
https://github.com/nasa/cumulus/pull/3126/files#diff-624123a7dc5bb5513e3769b27fe85821577087609a9376829676b7edbeb3a5aeR120
I know it's existing code, but does pgGranule already have collection_cumulus_id? 

Can we get all granules with the given `granule_id`?  and then check the collection_cumulus_id of the returned granules, and return error accordingly?  This way, we need one pg call from line 118 to 137 instead of 3

Reply:
searchByCumulusIds can get all the granules for a given granule cumulus_id.
https://github.com/nasa/cumulus/pull/3126/files#diff-6307984413f5b3418c109453a08f069ff55f4c3b68430454c19e96c714719f8cR186

Reply:
Yes, pgGranule has `collection_cumulus_id` so I removed the call to look that up here https://github.com/nasa/cumulus/pull/3126/commits/5a4e15cd762ac13eb519fd86e8fb43f27c3c8090

Reply:
https://github.com/nasa/cumulus/pull/3126/commits/1f3f24ca1830ed083f353610749919602031cc93